autobean_format/main.py

```diff
@@ -42,8 +43,11 @@ def load_files(self, filename: str) -> Iterator[_File]:
             if filename in visited:
                 continue
             visited.add(filename)
-            with open(filename) as f:
-                text = f.read()
+            lines = []
+            with fileinput.input(files=(filename)) as f:
```

Comment:
Redundant noop parentheses: `(filename)`

(I guess the intention is singleton tuple but looks like single filename is also acceptable here)